tests/func/test_diff.py

```diff
@@ -192,3 +192,39 @@ def setUp(self):
     def test(self):
         with self.assertRaises(FileNotInCommitError):
             self.dvc.diff(self.a_ref, target=self.unknown_file)
+
+
+class TestDiffModifiedFile(TestDiff):
```

Comment:
It would be great to convert `TestDiff` class to pytest fixture. Though it is not necessary in this change.

Reply:
noted